spec/views/user_index_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe 'User pages test index/show ', type: :feature do
+  before(:all) do
+    @first_user = User.create(name: 'Solomon', photo: 'https://avatars.githubusercontent.com/u/101924220?v=4',
+                              bio: 'test for User')
+    @post1 = Post.create(title: 'Rspec test 1', text: 'rspec test for post', user_id: @first_user)
+    @post2 = Post.create(title: 'Rspec test 2', text: 'rspec test for post', user_id: @first_user)
+    @post3 = Post.create(title: 'Rspec test 3', text: 'rspec test for post', user_id: @first_user)
+    @post4 = Post.create(title: 'Rspec test 4', text: 'rspec test for post', user_id: @first_user)
+  end
+
+  describe 'User index page test' do
+    it 'Should show the username ' do
+      visit users_path
+      expect(page).to have_content(@first_user.name)
+    end
```

Comment:
- [ ] Great job adding the integration specs. The index action normally displays all users. In this light and as per project requirement, there's no test to show username. Rather, there is a test to show username of all users which you have from lines 19 to 22. Remember to always use the DRY (Do not Repeat Yourself) principle. I think this particular test is not very necessary and a repetition.